src/inference/include/openvino/runtime/common.hpp

```diff
@@ -55,4 +55,23 @@ namespace runtime {
 using ov::SupportedOpsMap;
 }  // namespace runtime
 
-}  // namespace ov
\ No newline at end of file
+}  // namespace ov
+
+namespace std {
```

Comment:
Dose it work with out `std` namespace population?

Reply:
@apankratovantonp no